english/watagatapistsusberry/watagatapitsusberry_Fruit.rtf

```diff
@@ -0,0 +1 @@
+Berry with cranberry, you're feeling so hairy, maybe you're spaghetti? Those are the side effects of eating the watagatapitsusberry fruit! 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.